styles/modules.css

```diff
@@ -67,7 +67,9 @@
   margin-top: .2em;
 }
 
-@media only screen and (min-width: 690px) {
+/* Put in a break at 860 for header? */
+
```

Comment:
Nope! Changed the max-width in the 690px breakpoint to 48%, and that seems to keep things from overlapping, at least.